### PR TITLE
gnrc_rpl: make trickle parameters configurable

### PR DESCRIPTION
**Pull request by cgundogan**
Would this be a sufficient solution for #5061 ?

### CODE REVIEW

**Comment by BytesGalore** [comment]
Looks great :+1:  and enables to completely configure the intervals, ACK from my side for the change.
But @gebart must decide.

**Comment by gebart** [comment]
Looks good, will test.
If I change these parameters on the RPL border router then all other nodes in the DODAG are supposed to use the BR params, right?

**Comment by cgundogan** [comment]
>If I change these parameters on the RPL border router then all other nodes in the DODAG are supposed to use the BR params, right?

yeah correct. All nodes take these values from the DODAG Conf option that resides within the DIO.

File: sys/include/net/gnrc/rpl.h [hunk 1]
```diff
@@ -32,6 +32,13 @@
  *   CFLAGS += -DGNRC_RPL_WITHOUT_PIO
  *   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  *
+ *  - Modify trickle parameters
+ *   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ {.mk}
+ *   CFLAGS += GNRC_RPL_DEFAULT_DIO_INTERVAL_DOUBLINGS=20
```
Comment (gebart) [inline sys/include/net/gnrc/rpl.h]: CFLAGS += -Dxxxx=yyyy

**Comment by cgundogan** [comment]
@gebart thanks for the hint - I addressed your comment and amended immediately.

**Comment by gebart** [comment]
ACK, tested with modified border router makefile

**Pull request merged**